src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -449,28 +447,83 @@ class MacroAssembler: public Assembler {
   void fsflagsi(Register Rd, unsigned imm);
   void fsflagsi(unsigned imm);
 
-  void beqz(Register Rs, const address &dest);
-  void bnez(Register Rs, const address &dest);
-  void blez(Register Rs, const address &dest);
-  void bgez(Register Rs, const address &dest);
-  void bltz(Register Rs, const address &dest);
-  void bgtz(Register Rs, const address &dest);
-  void la(Register Rd, Label &label);
-  void la(Register Rd, const address &dest);
-  void la(Register Rd, const Address &adr);
+  // Control transfer pseudo instructions
+  void beqz(Register Rs, const address dest);
+  void bnez(Register Rs, const address dest);
+  void blez(Register Rs, const address dest);
+  void bgez(Register Rs, const address dest);
+  void bltz(Register Rs, const address dest);
+  void bgtz(Register Rs, const address dest);
+
+  void j(Label &l, Register temp = t0);
+  void j(const address dest, Register temp = t0);
+  void j(const Address &adr, Register temp = t0);
+  void jal(Label &l, Register temp = t0);
+  void jal(const address dest, Register temp = t0);
+  void jal(const Address &adr, Register temp = t0);
+  void jal(Register Rd, Label &L, Register temp = t0);
+  void jal(Register Rd, const address dest, Register temp = t0);
+
   //label
   void beqz(Register Rs, Label &l, bool is_far = false);
   void bnez(Register Rs, Label &l, bool is_far = false);
   void blez(Register Rs, Label &l, bool is_far = false);
   void bgez(Register Rs, Label &l, bool is_far = false);
   void bltz(Register Rs, Label &l, bool is_far = false);
   void bgtz(Register Rs, Label &l, bool is_far = false);
+
+  void beq(Register Rs1, Register Rs2, Label &L, bool is_far = false);
+  void bne(Register Rs1, Register Rs2, Label &L, bool is_far = false);
+  void blt(Register Rs1, Register Rs2, Label &L, bool is_far = false);
+  void bge(Register Rs1, Register Rs2, Label &L, bool is_far = false);
+  void bltu(Register Rs1, Register Rs2, Label &L, bool is_far = false);
+  void bgeu(Register Rs1, Register Rs2, Label &L, bool is_far = false);
```

Comment:
Do you want to line things like these up? For example:

```

  void beq (Register Rs1, Register Rs2, Label &L, bool is_far = false);
  void bne (Register Rs1, Register Rs2, Label &L, bool is_far = false);
  void blt (Register Rs1, Register Rs2, Label &L, bool is_far = false);
  void bge (Register Rs1, Register Rs2, Label &L, bool is_far = false);
  void bltu(Register Rs1, Register Rs2, Label &L, bool is_far = false);
  void bgeu(Register Rs1, Register Rs2, Label &L, bool is_far = false);
```

Reply:
Yes, that looks better. Fixed.